src/Sentry.Unity/SentryUnity.cs

```diff
@@ -51,10 +51,27 @@ public static void Init(SentryUnityOptions options)
                             "acquire a lockfile on the config directory: .NET event cache will be disabled.", ex);
                         options.CacheDirectoryPath = null;
                         options.AutoSessionTracking = false;
+
                     }
                 }
 
                 var sentryDotNet = SentrySdk.Init(options);
+
+                if (options.AttachScreenshot)
+                {
+                    if (ApplicationAdapter.Instance.Platform is RuntimePlatform.WebGLPlayer)
```

Comment:
Just to make sure: I thought we wanted to flat out disable screenshots inside Configure with an appropriate log?

Reply:
That's what happens here... Am I missing something?

Reply:
I was thinking in here: https://github.com/getsentry/sentry-unity/blob/242dd3acac6248a1cc27047ef71041f992893d94/src/Sentry.Unity/WebGL/SentryWebGL.cs#L14

Reply:
I think it'd fit nicely with future things like: https://github.com/getsentry/sentry-unity/issues/231
I want to get rid of the SentryOptionsUtility and move that stuff into its respective platform-specific Configure.
Maybe that's just me nitpicking.
LGTM! Thanks!